packages/monaco/src/browser/monaco-editor.ts

```diff
@@ -447,12 +447,20 @@ export class MonacoEditor implements TextEditor {
         this.editor.restoreViewState(state as monaco.editor.ICodeEditorViewState);
     }
 
+    /* `true` because it is derived from an URI during the instantiation */
+    protected _languageAutoDetected = true;
+
+    get languageAutoDeteceted(): boolean {
```

Comment:
typo, fix: `languageAutoDetected`

Reply:
oh, sorry, only noticed it, could you open a follow-up PR, it won't be a breaking change